zkevm-circuits/src/bytecode_circuit/circuit.rs

```diff
@@ -272,15 +276,17 @@ impl<F: Field> SubCircuitConfig<F> for BytecodeCircuitConfig<F> {
                 meta.query_advice(length, Rotation::cur()),
             );
 
-            let empty_hash = rlc::expr(
-                &EMPTY_CODE_HASH_LE.map(|v| Expression::Constant(F::from(v as u64))),
-                challenges.evm_word(),
-            );
+            let empty_hash_word: Word<Expression<F>> =
+                Word32::new(EMPTY_CODE_HASH_LE.map(|v| Expression::Constant(F::from(v as u64))))
+                    .to_word();
```

Comment:
We don't need to convert to 32 limbs of expressions and then merge to 2 limbs.
```suggestion
            let empty_hash_word: Word<Expression<F>> = Word::from(EMPTY_CODE_HASH).to_word();
```

Reply:
Word::from(CodeDB::empty_code_hash()) will produce a Word<Value<F>> not a Word<Expression<F>>.

Reply:
Do you want me to implement to_word trait for Word<Value<F>>? How should it behave if Value::unknown?